packages/workflow/src/utils.ts

```diff
@@ -1,34 +1,38 @@
 /* eslint-disable @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-argument */
-export const deepCopy = <T>(source: T, hash = new WeakMap(), path = ''): T => {
-	let clone: any;
-	let i: any;
+type Primitives = string | number | boolean | bigint | symbol;
```

Comment:
aren't `null` and `undefined` needed here?

Reply:
yes, added back and another improvement